client.go

```diff
@@ -107,7 +109,28 @@ func (cs *clientSession) run() (err error) {
 	cs.dc.OnOpen(cs.dataChannelOnOpen())
 	cs.dc.OnMessage(cs.dataChannelOnMessage())
 
-	if cs.offer, err = sd.Decode(cs.offerString); err != nil {
+	offerText := cs.offerString
+	if len(cs.offerString) == 32 {
+		url := fmt.Sprintf("https://www.10kb.site/%s", cs.offerString)
```

Comment:
this logic is now repeated in a few places, abstract to simple functions for setting and getting values?